roles/testnode/tasks/main.yml

```diff
@@ -64,3 +64,8 @@
 - include: pip.yml
   tags:
     - pip
+
```

Comment:
Could we add a comment here to indicate why we need this?  I know why, but the next person might not.  The commit log might also be a good place to explain the 'why' of this.